### PR TITLE
Feat/sidebar

### PR DESCRIPTION
**Pull request by cinaaaa**
The toolbar and sidebar are ready now.

Screenshot 2023-05-22 at 16 47 12
Screenshot 2023-05-22 at 16 47 20

There were some problems and not-clear thoughts, I've just opened some discussions with Ana on this and also some of the other things depend on the components that we still didn't make it though

## Obstacles

- The input that should be created in the future, as I tested the current text field it wasn't suitable to easily make the current input text field style in the Figma
- The goat icon was not grouped and I wasn't able to export it ( just asked Ana for this )
- Icon colors are not dynamic on props so I keep the icons color like this ( They're the wrong colors so should be changed in the future )

### CODE REVIEW

**Comment by majkshkurti** [comment]
Thanks Sina. 

> The input that should be created in the future, as I tested the current text field it wasn't suitable to easily make the current input text field style in the Figma

You can leave this for now. We can have a loot together. 

> The goat icon was not grouped and I wasn't able to export it ( just asked Ana for this )

Sure. Although as I mentioned in the review for the toolbar the GOAT icon should come from the GOAT app. You can get the svg from live version. Should be here: 
https://goat-dev.plan4better.de/img/goat_standard.svg
I also have it wrapped as a react component inside ui/assets/svg/GOATLogo.svg  but we shouldn't use it directly inside ui/components. 

> Icon colors are not dynamic on props so I keep the icons color like this ( They're the wrong colors so should be changed in the future )

This is the part that might be a bit confusing for now. 
The colors that are defined in the storybook color palette are the correct ones. The ones that you see in the Figma Ana will have to change. We should use P4B brand colors instead.  These colors are hardcoded in the packages/ui and they will not change across all apps.

File: app/client-v2/packages/ui/components/Toolbar.tsx
[the remark this answers is not in the document]
Comment (cinaaaa) [reply app/client-v2/packages/ui/components/Toolbar.tsx:1]: @majkshkurti  On this atomic design concept, I think we should have also the molecules, and organisms alongside the atoms, where this kind of toolbar or sidebar can be a generic & re-usable component and we could pass props in the molecules.

File: app/client-v2/packages/ui/components/Toolbar.tsx [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+  const { items } = props;
+  const { classes, cx } = useStyles();
+
+  return (
+    <Stack
+      className={cx(classes.root)}
+      direction="row"
+      justifyContent="space-between"
+      alignItems="center"
+      padding="21.5px">
```
[the remark this answers is not in the document]
Comment (cinaaaa) [reply app/client-v2/packages/ui/components/Toolbar.tsx:28]: > We should rather use theme.spacing instead

I often find myself utilizing `theme.spacing` for consistent margins and paddings in Figma. However, I've noticed that the numerical values for margins vary, with instances of 15px, 16px, or even 17px appearing in different places. As a result, the spacing ratios end up being irregular, such as `1.2` or `3.9`, which is far from ideal. Additionally, I've observed that the spacing tends to change across different breakpoints, whereas in our specific case, the padding should remain consistent regardless of size.

[the remark this answers is not in the document]
Comment (majkshkurti) [reply app/client-v2/packages/ui/components/Toolbar.tsx:28]: I understand. Then we have to define this with Ana as discussed.

File: app/client-v2/packages/ui/components/Toolbar.tsx
[the remark this answers is not in the document]
Comment (majkshkurti) [reply app/client-v2/packages/ui/components/Toolbar.tsx:1]: Yeah, that's the idea.  Maybe we don't have to be 100% strict on this pattern but as much as we can keep generic the better.

[the remark this answers is not in the document]
Comment (cinaaaa) [reply app/client-v2/packages/ui/components/Toolbar.tsx:1]: I'm not sure about this Majk, let's see this 0 and 1, if we don't want to use a pattern let's just don't, or we're going to have lots of time refactoring again in the future by the current approach.

**Pull request merged**